docs/api/sql/Function.md

```diff
@@ -1475,3 +1475,24 @@ SELECT ST_ZMin(ST_GeomFromText('LINESTRING(1 3 4, 5 6 7)'))
 ```
 
 Output: `4.0`
+
+## ST_NDims
+
```

Comment:
1. Please add more elaboration and examples about dimension when the input has M or Z.
2. The output should be `3` integer not `3.0` double. Please correct me if I am wrong.
3. The doc should be sorted by the alphabetical order